### PR TITLE
VZ-5342 Update Nginx controller to use JSON log messages

### PR DESCRIPTION
**Pull request by davisgomes**
# Description

This updates the Nginx controller and the Fluentd config to handle JSON log messages and parse them correctly. 

Fixes VZ-5342

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml [hunk 1]
```diff
@@ -642,15 +642,9 @@ data:
       <parse>
         @type multi_format
         <pattern>
-          format /^(?:[^ ]*) (?:[^ ]*) (?:[^ ]*) \[(?<time>[^\]]*)\] "(?<method>\S+)(?: +(?<path>[^\"]*?)(?: +\S*)?)?" (?<code>[^ ]*) (?:[^ ]*)(?: "(?:[^\"]*)" "(?:[^\"]*)") (?<message>[\s\S]*)$/
-          time_key time
-          time_format %d/%b/%Y:%H:%M:%S %z
-        </pattern>
-        # Kubernetes klog format
-        <pattern>
```
Comment (bobdonat) [inline platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:650]: Should the klog format pattern remain

**Review by kminder — APPROVED** [review]
Approved.

**Comment by bobdonat** [comment]
Does the AT need to be uodated

File: platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml (hunk 1 above)
Comment (davisgomes) [reply platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:650]: I don't think so. I didn't see any klog formatted messages.

Comment (kminder) [reply platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:650]: Oh I overlooked this.  The klog messages are somewhat intermittent so you might not always see them.

**Review by bobdonat — APPROVED** [review]

**Pull request merged**